inference-engine/src/transformations/include/transformations/utils/utils.hpp

```diff
@@ -48,8 +48,15 @@ bool has_op_with_type(const std::shared_ptr<const ngraph::Function> &function) {
 inline std::string create_ie_output_name(const ngraph::Output<ngraph::Node>& output) {
     const auto& prev_layer = output.get_node_shared_ptr();
     std::string out_name = prev_layer->get_friendly_name();
-    if (prev_layer->get_output_size() != 1)
+    size_t num_outputs(0);
+    for (const auto& out : prev_layer->outputs()) {
+        if (!out.get_target_inputs().empty())
+            num_outputs++;
+    }
+    if (num_outputs > 1 || output.get_target_inputs().empty())
         out_name += "." + std::to_string(output.get_index());
+    if (output.get_target_inputs().empty())
+        out_name += ".fake";
```

Comment:
why it's fake? it just not used output. Why we can not use index instead of 'fake'?

Reply:
I just wanted to mark these outputs. Because if we just add indexes in can be confused.
Do you have another ideas?

Reply:
And looks like this statement ca override previous statement because they both return true for `output.get_target_inputs().empty()`

Reply:
Fixed